### ISSUE TITLE
feat: users password reset in the admin panel

### ISSUE THREAD

**Issue by KshitizSharmaV** [ollama-webui/ollama-webui#390]
**Is your feature request related to a problem? Please describe.**
Currently, there's no option in the graphical user interface to reset passwords for other users, which is a significant inconvenience. As an admin, I find it frustrating to resort to manual interventions in the database for such a common task. This process is not only time-consuming but also increases the risk of errors and security vulnerabilities.

**Describe the solution you'd like**
I propose implementing a feature in the admin panel that allows administrators to reset passwords for other users. This functionality should be straightforward to use, providing a secure and efficient way for admins to manage user accounts and maintain system security.

**Describe alternatives you've considered**
The current workaround involves directly accessing and modifying the database to reset user passwords. While this method is functional, it is not ideal due to its complexity and the potential risks associated with manual database operations. A more user-friendly and safer alternative within the admin interface is much needed.

**Additional context**
Implementing this feature would greatly enhance the admin panel's functionality and user management efficiency. It would also align with best practices for system administration and user account security. I believe this improvement would be a valuable addition to the system for all administrators. Thank you for considering this request.

**Comment by KshitizSharmaV** [comment]
I will try to pitch in for this feature. 
Thanks

**Comment by tjbck** [comment]
Hi, Thanks for the suggestion! I'll actively start working on this, stay tuned!

**Comment by justinh-rahb** [comment]
Lol yep I've already got a user that forgot their password, fortunately they had already downloaded their chat history from the user login migration so I just deleted the account, got them to sign up again, and reimport their history. A password reset function would be helpful :)

Can I suggest that this work maybe coincide with adding SMTP email support? That way we could send password reset links, and it could also be used for initial sign-up email verification, or inviting users with a registration link.
